docs/src/tutorials/supported_operations.md

```diff
@@ -169,13 +169,21 @@ The needs to be readable by `thin-edge.io` user - `tedge` - and should have perm
 
 In this example we want `thin-edge.io` to pick up a message on specific topic and execute the command on the device, our topic is `c8y/s/ds`.
 We also know that the message we expect is going to use SmartRest template `511` and our plugin is located in `/etc/tedge/operations/command`.
+The operation is configured to `timeout` after 10 seconds.
+```admonish note
+To avoid a custom operation from being run for longer, the custom operation can be configured to `timeout` after some time.
```

Comment:
I'd drop this line if the above suggestion is accepted.

Reply:
done